plugins/generator-1.19.4/forge-1.19.4/procedures/entity_data_integer_get.java.ftl

```diff
@@ -0,0 +1 @@
+(((${generator.map(field$customEntity, "entities", 0)}) ${input$entity}).getEntityData().get(${generator.map(field$customEntity, "entities", 0)}.${field$accessor}))
```

Comment:
What happens if the entity passed is not of the correct type? Does the game directly crash?

Reply:
MC does crash when it encounters a NPE, CCE or other minor problems like that.

Do you mean it should use a pattern variable on one branch and type default value on another (be written like `a inst_of b c ? c.get(b.d) : 0` instead of `((b) a).get(b.d)`)?

Reply:
I would add instanceof checks and return -1 otherwise or not set value if not the right type, similar to how this is handled eg. if a certain block does not have inventory or energy or fluid tank and such procedure block is operated on it.

It is hard to expect from beginners and people with no programming experience to predict all those cases so MCreator safeguards those cases when possible

Reply:
Did I use `${cbi}` correctly?

Reply:
I don't think setters need it

Reply:
This should also be handled somehow:

![image](https://github.com/MCreator/MCreator/assets/16374228/15db4aa2-ad0f-4d30-8fef-b774aff82fc5)